GitUI/CommandsDialogs/FormBrowse.Designer.cs

```diff
@@ -1741,13 +1744,14 @@ private void InitializeComponent()
             // 
             // FormBrowse
             // 
-            this.AutoScaleDimensions = new System.Drawing.SizeF(96F, 96F);
+            this.AutoScaleDimensions = new System.Drawing.SizeF(144F, 144F);
```

Comment:
❗️  All UI changes must be done at 100% scale factor.

VS typically prompts to do that
![image](https://user-images.githubusercontent.com/4403806/59270744-9c314080-8c95-11e9-81a8-dd51ae677334.png)


Please run VS with `/noscale` switch 
![image](https://user-images.githubusercontent.com/4403806/59431442-4d67e000-8e28-11e9-91e0-029b2dbc8e9f.png)

Reply:
❗️  All UI changes must be done at 100% scale factor.

Please refer to https://github.com/gitextensions/gitextensions/wiki/Guidelines%3A-UI-design